java/runtime/src/main/java/io/ray/runtime/serializer/MessagePackSerializer.java

```diff
@@ -83,6 +89,11 @@
           packer.packBinaryHeader(bytes.length);
           packer.writePayload(bytes);
         }));
+    packers.put(ObjectRefImpl.class, (object, packer, javaSerializer) -> {
+        byte[] payload = ((ObjectRefImpl<?>) object).toCrossLanguageBytes();
+        packer.packExtensionTypeHeader(OBEJECT_REF_TYPE_EXTENSION_ID, payload.length);
+        packer.addPayload(payload);
+    });
```

Comment:
Since we have ARRAY_PACKER as well, would this allow returning a list of object refs?